server/channels/jobs/s3_path_migration/s3_path_migration.go

```diff
@@ -110,7 +111,9 @@ func (worker *S3PathMigrationWorker) DoJob(job *model.Job) {
 	}
 
 	if worker.fileBackend == nil {
-		mlog.Error("S3PathMigrationWorker: no s3 file backend found.")
+		err := errors.New("no S3 file backend found")
+		mlog.Error("S3PathMigrationWorker: ", mlog.Err(err))
```

Comment:
For future reference, does setting a job error not log that error somewhere already?

Reply:
Setting a job error logs the error in the DB. But to log it in the normal log, you have to explicitly call `mlog.Error`.

Reply:
Ah, interesting. I would have expected a job failure to log an error for all jobs. TIL!